### PR TITLE
remove margin

### PR DESCRIPTION
**Pull request by tomltd**

### CODE REVIEW

File: src/css/style.css [hunk 1]
```diff
@@ -302,7 +302,6 @@ button.nav-link:active {
 
 [aria-label$="menu"] li {
   display: inline-block;
-  margin: 0.25em;
 }
```
Comment (katjam) [inline src/css/style.css]: Why?

**Comment by tomltd** [comment]
Otherwise the buttons spill over to two lines. With no margin which is tiny anyway they fit on one line. Not with ##MISSING COPY## though :)

**Comment by katjam** [comment]
OK - spill over on 2 lines at what screen size? And with what text... both
variable/ subject to change.

Yeah - sorry about the missing copy bit but the text needs to change -
hopefully to something shorter.

**Comment by tomltd** [comment]
can deal with this after copy as it will change

**Pull request closed without merging**